ivy/functional/backends/torch/statistical.py

```diff
@@ -1,8 +1,8 @@
 # global
+from torch import Tensor
```

Comment:
I think this is redundant, as torch is already imported you can directly use `torch.Tensor`. I also don't seem to find any usage of `Tensor` instead of `torch.Tensor` in your code.